Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py

```diff
@@ -0,0 +1,611 @@
+        if SNXReportParser.JSON_FILE_MODIFIED in response.keys():
+            jsonContent[JsonTableParser.SNX_FILE_MODIFIED] = response[SNXReportParser.JSON_FILE_MODIFIED]
+        if SNXResponse.SNX_STATUS_KEY in response.keys():
+            jsonContent[SNXResponse.SNX_STATUS_KEY] = response[SNXResponse.SNX_STATUS_KEY]
+        if SNXResponse.SNX_MSG_KEY in response.keys():
+            jsonContent[SNXResponse.SNX_ERROR_MSG_KEY] = response[SNXResponse.SNX_MSG_KEY]
+        return jsonContent
+
+    except Exception as e:
+        raise DemistoException(e)
+
+
+def format_report_contents(contents):
+    try:
+        def dict_to_string(nested_dict):
+            return json.dumps(nested_dict).lstrip('{').rstrip('}').replace('\'', '').replace('\"', '')
+
+        table_contents = OrderedDict()
+        for key, val in contents.items():
+            if isinstance(val, dict):
+                table_contents[key] = dict_to_string(val)
+            elif isinstance(val, list):
+                table_values = []
+                for item in val:
+                    if isinstance(item, dict):
+                        table_values.append(dict_to_string(item))
+                    else:
+                        table_values.append(item)
+                table_contents[key] = table_values
+            else:
+                table_contents[key] = val
+        return table_contents
+    except Exception as e:
+        raise DemistoException(e)
+
+
+def post_submit_file(client: Client, args: Dict[str, str]) -> CommandResults:
+    urlSuffix = "/submit_file"
+    entryId = args.get('EntryID') or None
+    if entryId is None:
+        raise DemistoException("Entry ID Not Found")
+    platformValue = args.get('platform') or SNXResponse.SNX_WINDOWS_KEY
+    params = create_request_json(args)
+    if 'platform' not in params.keys():
+        params['platform'] = platformValue
+    fileEntry = demisto.getFilePath(entryId)
+    fileName = fileEntry['name']
+    filePath = fileEntry['path']
+    fileData = {'file': (fileName, open(filePath, 'rb'))}
+    response, err_msg = client.submit_file(urlSuffix, fileData, params)
+    if response:
+        if SNXResponse.SNX_SUCCESS_KEY in response.keys() and SNXResponse.SNX_RESULT_KEY in response.keys():
+            finalJson = response[SNXResponse.SNX_RESULT_KEY]
+            readableOutput = tableToMarkdown(f"File Submitted Successfully: {fileName}", finalJson)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="SnXAnalysis.SubmitFile",
+                outputs=finalJson
+            )
+        else:
+            readableOutput = tableToMarkdown("File Submission Failed: {fileName}", response)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="",
+                outputs=response
+            )
```

Comment:
This `outputs` will not be under a special key.
Consider removing `outputs` so it doesn't return to context, or add to `output_prefix` like this:

```suggestion
            return CommandResults(
                readable_output=readableOutput,
                outputs_prefix="SnXAnalysis.SubmitFile",
                outputs=response
            )
```